src/Schema/IntColumnDefinition.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
May I know the case of `declare(strict_types=1);` just for my information?

It is defined in `IntColumnDefinition` and not in `SequenceDefinition`.

Reply:
Newly added files have them. I forgot to add it to SequenceDefinition. Adding it now.